jbmc/src/java_bytecode/README.md

```diff
@@ -25,7 +25,80 @@ To be documented.
 
 \subsection java-class-section How a java program / class is represented in a .class
 
-To be documented.
+Every Java class is compiled into a .class file, inner classes, anonymous
```

Comment:
Clearer as for example "Every Java class is compiled into a .class file. Inner classes, anonymous classes or classes for tableswitches are also compiled into their own .class files."